File: 01 Introdução/3303 Palavrão/3303.java
import java.util.Scanner;
class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    String palavra = sc.nextLine();
    if (palavra.length() >= 10){
      System.out.println("palavrao");
    } else {
      System.out.println("palavrinha");
    }
    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/3303
// Ranking 00044º, Time 0.122, Submiss Date 16/04/2023 14:21:32 (Java 14)

File: 01 Introdução/3303 Palavrão/3303.c
# include <stdio.h>
# include <string.h>

int main(){
  char palavra[20];
  scanf("%s", palavra);
  if (strlen(palavra) >= 10){
    printf("palavrao\n");
  } else {
    printf("palavrinha\n");
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/3303
// Ranking 00454º, Time 0.000, Submiss Date 12/04/2023 10:37:20 (C99)